src/Generator/ShippingProfiles.php

```diff
@@ -126,65 +211,71 @@ private function row(array $row):array
     /**
      * Get list of supported shipping profile ids for the given item.
      *
-     * @param  Record $item
-     * @return array<int,int>
+     * @param  int $itemId
+     * @param  bool $shippingProfileName
+     * @return array
      */
-    private function getShippingSupportIds(Record $item):array
+    private function getShippingProfilesList(int $itemId, bool $shippingProfileName = false):array
```

Comment:
I've putted that option just for further development, because I thought it will be useful to have in the future the option to export the names. For what I know now, this could be removed, because the format is mostly used for mass exporting, external modification and then importing.